idb/idb_js.go

```diff
@@ -15,20 +15,24 @@ const (
 
 var (
 	evalOnce = sync.Once{}
+	openIDB  js.Value
+	idb      js.Value
 )
 
-func setup() {
-	js.Global().Call("eval", idbCode)
-	js.Global().Call("eval", `
-var replicache = {};
+func eval(code string) js.Value {
+	return js.Global().Call("eval", code)
+}
 
-replicache.openIDB = async function(dbname) {
-    return await idb.openDB(dbname, 1, {
-      upgrade(db) {
-          db.createObjectStore("chunks");
-        },
-    });
-};`)
+func setup() {
+	idb = eval(idbCode)
```

Comment:
eval returns the completion value. This allows us to evaluate the code without adding global variables.